### DISCUSSION TITLE
Non reconnaissance de disque externe

### DISCUSSION THREAD

**Discussion by LucieVantard** [elementary/triage#573] [Q&A]
J'ai un message d'erreur quand j’essaie d’accéder aux documents de mon disque dur externe
![Capture-Disque-externe-2024-09-11 13 32 13](https://github.com/user-attachments/assets/7ad118a6-b4bb-4ad6-bc88-559443eaf13d)

Je ne peux plus acceder au contenu. Comment faire ?

**Comment by jeremypw** [comment]
Is it a NTFS (Windows)  format disk?

**Reply by LucieVantard** [reply]
Non la marque est "bipra", il a déjà fonctionné sur mon ordinateur avec elementary

**Reply by ryonakano** [reply]
@LucieVantard Could you open the Terminal app, run `lsblk -f` command, and then copy and paste its output here? We want to see what is the format (file system) of your disk.
